### PR TITLE
LPS-191512 Move experience selector to Page Audit panel

### PR DESCRIPTION
**Pull request by veroglez**
This code depends on https://github.com/liferay-echo/liferay-portal/pull/13114

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-js-components-web/src/main/resources/META-INF/resources/js/components/experience_selector/ExperienceSelector.tsx [hunk 1]
```diff
@@ -24,7 +32,15 @@ const TriggerLabel = React.forwardRef(
 				ref={ref}
 			>
 				<Layout.ContentRow className="c-pr-3" verticalAlign="center">
-					<Layout.ContentCol className="text-truncate" expand>
+					<Layout.ContentCol
+						className={classNames(
+							'c-mr-2',
+							truncatedSelectorText
+								? 'text-truncate'
```
Comment (victorg1991) [inline modules/apps/layout/layout-js-components-web/src/main/resources/META-INF/resources/js/components/experience_selector/ExperienceSelector.tsx:52]: shouldn't we avoid truncating the text?

File: modules/apps/layout/layout-js-components-web/src/main/resources/META-INF/resources/js/components/experience_selector/ExperienceSelector.tsx [hunk 2]
```diff
@@ -118,9 +121,12 @@ export default function ExperienceSelector({
 									{item.segmentsExperienceName}
 								</Text>
 
+								<span className="sr-only">{`. ${getDescriptionLabel(
```
Comment (victorg1991) [inline modules/apps/layout/layout-js-components-web/src/main/resources/META-INF/resources/js/components/experience_selector/ExperienceSelector.tsx:124]: Why do we need a colon at the start of the sentence?

**Comment by victorg1991** [comment]
LGTM :) just a couple of question there, also it seems that there is a rebase error, could you check it out @veroglez ? thanks!!

File: modules/apps/layout/layout-js-components-web/src/main/resources/META-INF/resources/js/components/experience_selector/ExperienceSelector.tsx (hunk 2 above)
Comment (veroglez) [reply modules/apps/layout/layout-js-components-web/src/main/resources/META-INF/resources/js/components/experience_selector/ExperienceSelector.tsx:124]: This is for the screen reader. First reads the name of the experience, and the ". segment:Anyone, active". The colon is to separate the name with the rest of the text. The result is as follows:
Screen Shot 2023-08-04 at 10 14 57 AM

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Comment by veroglez** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward:force

**Pull request closed without merging**